### ISSUE TITLE
 Learn pathway is not working on Microsoft Teams

### ISSUE THREAD

**Issue by Narendraka** [pnp/custom-learning-office-365#599]
>**PROVISIONING NOTE**: If you are having issues with the SharePoint look book while trying to provision M365 learning pathways please do not post that issue to this repo as we cannot help you here. Please post all questions and issues for the SharePoint look book to [SharePoint/sp-provisioning-service GitHub Issues List](https://github.com/SharePoint/sp-provisioning-service/issues).

>**WEB PART NOT LOADING**: If you are getting the error: "Microsoft 365 learning pathways has a configuration issue. Ask your administrator for assistance". Please review the [Diagnosing Pre-Check](https://github.com/pnp/custom-learning-office-365/blob/main/installation/DiagnosisPreCheck.md) guidance before submitting your bug.

### To Reproduce

>Steps to reproduce the behavior

### Expected behavior

>A clear and concise description of what you expected to happen.

### Screenshots

>If applicable, add screenshots to help explain your problem.

### Learning Pathways version number

>Please share the version number of learning pathways in your tenant app catalog.

### Additional context

>Add any other context about the problem here.

**Comment by dcashpeterson** [comment]
@Narendraka Thanks for submitting this issue. To help you further we will need additional information. If you could please fill out the issue template with that information it would help us understand where this is coming from. Also, if you could include any errors from the browser console.

**Comment by Narendraka** [comment]
> @Narendraka Thanks for submitting this issue. To help you further we will need additional information. If you could please fill out the issue template with that information it would help us understand where this is coming from. Also, if you could include any errors from the browser console.

**Comment by Narendraka** [comment]
@dcashpeterson can you please in provide issue template, this app published for all user in org and when users try accessing the app its giving attached error, same issue on Teams web as well

**Comment by Narendraka** [comment]
@v-harishd can you please update if you got any info on this

**Comment by amitkh22** [comment]
@Narendraka To help you further we will need additional information. You can go through **Diagnosis Pre-Check** guidance (https://github.com/pnp/custom-learning-office-365/blob/main/installation/DiagnosisPreCheck.md) and share any errors messages from browser console

**Comment by dcashpeterson** [comment]
@Narendraka Here is the issue template. We will need this filled out before we can assist. As @amitkh22 said, please check the diagnoisi pre check first but we will also need the information in the template filled out. Without that we have no way of being able to assist you.

Thanks

>**PROVISIONING NOTE**: If you are having issues with the SharePoint look book while trying to provision M365 learning pathways please do not post that issue to this repo as we cannot help you here. Please post all questions and issues for the SharePoint look book to [SharePoint/sp-provisioning-service GitHub Issues List](https://github.com/SharePoint/sp-provisioning-service/issues).

>**WEB PART NOT LOADING**: If you are getting the error: "Microsoft 365 learning pathways has a configuration issue. Ask your administrator for assistance". Please review the [Diagnosing Pre-Check](https://github.com/pnp/custom-learning-office-365/blob/main/installation/DiagnosisPreCheck.md) guidance before submitting your bug.

## Describe the bug

A clear and concise description of what the bug is.

### To Reproduce

>Steps to reproduce the behavior

### Expected behavior

>A clear and concise description of what you expected to happen.

### Screenshots

>If applicable, add screenshots to help explain your problem.

### Learning Pathways version number

>Please share the version number of learning pathways in your tenant app catalog.

### Additional context

>Add any other context about the problem here.

**Comment by dcashpeterson** [comment]
Closed due to inactivity. If this is still an issue please create a new ticket and reference this one.
